components/thumbnail/index.css

```diff
@@ -1,4 +1,4 @@
-/*!
+/*
```

Comment:
Just wanted to flag this change - be sure to bring back the /*! before merging. It's a new standard we're using to ensure copyrights don't get stripped